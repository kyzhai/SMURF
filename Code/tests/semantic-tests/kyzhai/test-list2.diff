1,2c1,2
< *** tx: Num  ty: Num
< Fatal error: exception Sast.Type_error("1 has type of Num but true has type Bool in a same list")
---
> *** tx: Int  ty: Int
> Fatal error: exception Sast.Type_error("1 has type of Int but true has type Bool in a same list")
