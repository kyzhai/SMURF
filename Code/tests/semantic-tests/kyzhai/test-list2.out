*** tx: Num  ty: Num
Fatal error: exception Sast.Type_error("1 has type of Num but true has type Bool in a same list")
